lib/exprotobuf.ex

```diff
@@ -60,11 +60,13 @@ defmodule Protobuf do
   defp parse(%Config{namespace: ns, schema: schema, inject: inject, from_file: nil}, _) do
     Parser.parse!(schema) |> namespace_types(ns, inject)
   end
-  defp parse(%Config{namespace: ns, schema: schema, inject: inject, from_file: file}, caller) do
-    {path, _} = Code.eval_quoted(file, [], caller)
-    path      = Path.expand(path) |> Path.dirname
-    opts      = [imports: [path]]
-    Parser.parse!(schema, opts) |> namespace_types(ns, inject)
+  defp parse(%Config{namespace: ns, inject: inject, from_file: file}, caller) do
+    {paths, import_dirs} = resolve_paths(file, caller)
```

Comment:
this is pretty much the core of my changes. This supports passing in either a single path or a list of paths that the user wants to compile. Then we parse each of those files and pass the results of all of the parses into `Builder`